app/assets/javascripts/scheduler/store.js

```diff
@@ -48,18 +41,25 @@ export const defaultState = (
   };
 
   const time_values = populateTimes(LOCALE === "en" ? "off" : "on");
-  const timeValuesTodayLeft = timeValuesToday([yearMonthDay(firstDay)], time_values);
+  let timeValuesTodayLeft = timeValuesToday([yearMonthDay(firstDay)], time_values);
+
   let selectedTimeValue = null;
+  // Select first available time slot only if there are some left for the 
+  // current day, i.e. hour is less than 23h00.
   if (timeValuesTodayLeft && timeValuesTodayLeft.length > 0) {
     selectedTimeValue = timeValuesTodayLeft[0].val;
-  } 
+  }
+  // If there is no time slots remaining in the current day, we are past
+  // 23h00 and we need to shift forward the first available day for scheduling. 
   else {
-    // REVIEW: Raises error past 23h00m in a day, not great when debugging late.  
-    // TODO:This is not the proper fix and could lead to bugs.
-    //      This is a partial fix as this crashes the whole page past 11pm.
+    const tomorrow = dayjs().add(1, 'day').startOf("date");
```

Comment:
Can this be written as 
```suggestion
    const tomorrow = today.add(1, 'day');
```

or 

```suggestion
    const tomorrow = today.add(1, 'day').startOf("date");
```
?

Reply:
No, `today` is carried in the context and that function called multiple times, hence it will be incremented several times, leading to push forward the first available day. This is a edge case I had to deal with as I wrote it this way initially.